src/auto-generator.ts

```diff
@@ -660,15 +661,15 @@ export class AutoGenerator {
   }
 
   private isNumber(fieldType: string): boolean {
-    return /^(smallint|mediumint|tinyint|int|bigint|float|money|smallmoney|double|decimal|numeric|real)/.test(fieldType);
+    return /^(smallint|mediumint|tinyint|int|bigint|float|money|smallmoney|double|decimal|numeric|real|date|time)(?:\(|$)/.test(fieldType);
```

Comment:
It looks like you are representing DATE and TIME types as `number` instead of `string`,

Reply:
Whoops! I'll swap that